plugin/builtin/helmchartinflationgenerator/HelmChartInflationGenerator.go

```diff
@@ -287,6 +287,9 @@ func (p *HelmChartInflationGeneratorPlugin) templateCommand() []string {
 	if p.IncludeCRDs {
 		args = append(args, "--include-crds")
 	}
+	if p.SkipHooks {
+		args = append(args, "--no-hooks")
```

Comment:
Is it possible to add a test for this in `HelmChartInflationGenerator_test.go`?

Reply:
Sure! I've added a test.